CHANGELOG.md

```diff
@@ -1,5 +1,133 @@
 # Changelog
 
+## 0.38.0
+
+**Release date:** 2024-05-06
+
+This prerelease graduates the `ImageUpdateAutomation` API to v1beta2.
+
+### `image.toolkit.fluxcd.io/v1beta2`
+
+After upgrading the controller to v0.38.0, please update the
+`ImageUpdateAutomation` **Custom Resources** in Git by replacing
+`image.toolkit.fluxcd.io/v1beta1` with `image.toolkit.fluxcd.io/v1beta2` in all
+YAML manifests.
+
```

Comment:
Bumping the API version in manifests can be done gradually. It is advised not to delay this procedure as the v1beta1 version will be removed after 6 months.